datasets/fr/tresor/fr_tresor_gels_avoir.yml

```diff
@@ -4357,3 +4357,191 @@ lookups:
           address: "676811 Amurskaya oblast, Belogorskiy Rayon, Vozhaevka"
           innCode: 2811005250
+      - match: "Type d'entité: société par actions: /"
+        props:
+          legalForm: société par actions
+      - match: "numéro d'identification fiscale: 9718077239"
+        props: 
+          innCode: 9718077239
+      - match: "Lieu d'enregistrement: Fédération de Russie (107241, Moscou, sh. Shchelkovskoe, 23A, pomeshch. V kom. 33.) - Date d'enregistrement: 27.9.2017 - Principal établissement: Fédération de Russie: /"
+        props:
+          jurisdiction: ru
+          address: "107241, Moscou, sh. Shchelkovskoe, 23A, pomeshch. V kom. 33."
+          incorporationDate: 2017-09-27
+      - match: "Numéro d’enregistrement: 1027810239555"
+        props:
+          registrationNumber: 1027810239555
+      - match: "Type d'entité: société par actions. Lieu d'enregistrement: 127287, st. Khutorskaya 2nd, 38A, Moscou, Fédération de Russie - Date d'enregistrement: 28.11.2002 - Principal établissement: Fédération de Russie: /"
+        props:
+          jurisdiction: ru
+          address: "127287, st. Khutorskaya 2nd, 38A, Moscou"
+          legalForm: société par actions
+          incorporationDate: 2002-11-28
+      - match: "Numéro d’enregistrement: 1027739642281"
+        props:
+          registrationNumber: 1027739642281
+      - match: "Type d'entité: société publique par actions - Lieu d'enregistrement: 107078, Moscou, Masha Poryvaeva str., 34, Fédération de Russie - Date d'enregistrement: 2.3.1993 - Principal établissement: Fédération de Russie: /"
+        props:
+          jurisdiction: ru
+          address: "107078, Moscou, Masha Poryvaeva str., 34"
+          legalForm: société publique par actions
+          incorporationDate: 1993-03-02
+      - match: "OGRN: 1027739460737"
+        props:
+          ogrnCode: 1027739460737
+      - match: "Numéro d’enregistrement: 1057746595367"
+        props:
+          registrationNumber: 1057746595367
+      - match: "Numéro d’identification fiscale: 7704552473"
+        props:
+          innCode: 7704552473
+      - match: "Numéro d’enregistrement: 1037719005873"
+        props:
+          registrationNumber: 1037719005873
+      - match: "OGRN: 1207700135972"
+        props:
+          ogrnCode: 1207700135972
+      - match: "INN: 9706005582"
+        props:
+          innCode: 9706005582
+      - match: "OGRN: 1087746736296"
+        props:
+          ogrnCode: 1087746736296
+      - match: "INN: 7705846236"
+        props:
+          innCode: 7705846236
+      - match: "Numéro d’enregistrement: 1020300887793"
+        props:
+          registrationNumber: 1020300887793
+      - match: "Numéro d’enregistrement: 1021603617188"
+        props:
+          registrationNumber: 1021603617188
+      - match: "Type d'entité: société par actions: /"
+        props:
+          legalForm: société par actions
+      - match: "Numéro d'enregistrement : 1065471010340"
```

Comment:
just take another look at the console output. It looks like some of them aren't working. In this case I see two spaces in the console output.

`Numéro d'enregistrement  : 1065471010340`